File: app/src/main/java/net/osdn/gokigen/blecontrol/lib/ble/connect/ITextDataUpdater.java
package net.osdn.gokigen.blecontrol.lib.ble.connect;

public interface ITextDataUpdater
{
    void setText(String data);
    void addText(String data);
    void showSnackBar(String message);
    void showSnackBar(int rscId);

    void enableOperation(boolean isEnable);
}

File: app/src/main/java/net/osdn/gokigen/blecontrol/lib/ble/connect/ITextDataUpdater.kt
package net.osdn.gokigen.blecontrol.lib.ble.connect

interface ITextDataUpdater {
    fun setText(data: String?)
    fun addText(data: String?)
    fun showSnackBar(message: String?)
    fun showSnackBar(rscId: Int)

    fun enableOperation(isEnable: Boolean)
}
